automation_tools/__init__.py

```diff
@@ -866,7 +866,7 @@ def setup_foreman_discovery(sat_version):
     )
     admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
 
-    if sat_version in ('6.1', '6.2', '6.3'):
+    if sat_version in ('6.1', '6.2'):
```

Comment:
you are correct. the 6.3 case is handled at
https://github.com/SatelliteQE/automation-tools/pull/651/files#diff-a692ae115cc45cc7627d4545e9b4c294R906
and it differs from the older versions. Great catch, btw.